packages/components/pie-radio-group/src/index.ts

```diff
@@ -11,10 +25,133 @@ const componentSelector = 'pie-radio-group';
 
 /**
  * @tagname pie-radio-group
+ * @slot - Default slot
+ * @slot label - The label slot
+ * @event {CustomEvent} change - when one of the radios state is changed.
  */
-export class PieRadioGroup extends LitElement implements RadioGroupProps {
+export class PieRadioGroup extends FormControlMixin(RtlMixin(LitElement)) implements RadioGroupProps {
+    @state()
+        hasLabel = false;
```

Comment:
```suggestion
    private _hasLabel = false;
```

Reply:
updated the checkbox group component too for consistency 👜